pyproject.toml

```diff
@@ -56,6 +56,7 @@ dependencies = [
   "typer>=0.7.0",
   "uvicorn>=0.23",
   "watchdog>=3.0.0",
+  "google-cloud-language>=2.14.0",
```

Comment:
This should be added as a separate extra, i.e. `pip install nemoguardrails[gcp]`.